tasks/run-matlab-build/v1/matlab.ts

```diff
@@ -0,0 +1 @@
+../../run-matlab-command/v1/matlab.ts
```

Comment:
How does this resolve, does it just copy the entire file contents to this location like including a header file?

Reply:
Yeah this is how GitHub shows symlinks. It makes it so we can reuse the run-command logic and not have it in each of the separate tasks like we did for v0. This has resulted in us not remembering to update run-command style logic in all tasks in the past.